### PR TITLE
Migrate docs

### LINKED ISSUE #1151: Migrate to mkdocs
Sphinx is nice but mkdocs is more fun and easier to maintain.
Check carps or amltk repo.

### PR DESCRIPTION
**Pull request by benjamc**
- Update windows install guide (#952)
- Migrate sphinx docs to mkdocs (#1155)
- Closes #1151

Run `make docs` to see new documentation page.

### CODE REVIEW

**Comment by LukasFehring** [comment]
Can we increase the contrast for code blocks? currently one can not see the different background color on first glance

**Comment by LukasFehring** [comment]
For the examples page it is confusing, that in the index column the chpoaters start with numbers.

**Comment by LukasFehring** [comment]
What do we need the `Exapnd to Copy` in the examples for?

**Comment by LukasFehring** [comment]
In API: What does the Constants Refer to (I did not find this in the code)?

**Review by LukasFehring — CHANGES_REQUESTED** [review]

**Comment by benjamc** [comment]
Examples page: numbers are needed to create the order, no other fix right now. In the code it is fine.
Expand to copy: does not hurt, might be more interesting if we have notebooks in the examples
Constants: added small docstring

File: docs/advanced_usage/3_multi_objective.md [hunk 1]
```diff
@@ -18,24 +17,24 @@ The basic recipe is as follows:
...
 
-   # Or you use the intensifier
-   incumbents = smac.intensifier.get_incumbents()
+  # Or you use the intensifier
+  incumbents = smac.intensifier.get_incumbents()
+```
 
-
-We show an example of how to use multi-objective with plots in our :ref:`examples<Multi-Objective>`.
+We show an example of how to use multi-objective with plots in our [examples](../../examples/3%20Multi-Objective/1_schaffer).
```
[the remark this answers is not in the document]
Comment (benjamc) [reply docs/advanced_usage/3_multi_objective.md]: fixed

File: docs/advanced_usage/5_ask_and_tell.md [hunk 2]
```diff
@@ -1,20 +1,19 @@
...
+!!! warning
 
     It might be the case that not all user-provided trials can be considered. Take Successive Halving, for example, 
     when specifying the min and max budget, intermediate budgets are calculated. If the user provided trials with
     different budgets, they, obviously, can not be considered. However, all user-provided configurations will flow 
     into the intensification process.
 
 
-Please have a look at our :ref:`ask-and-tell example<Ask-and-Tell>`.
+Please have a look at our [ask-and-tell example](../../examples/1%20Basics/3_ask_and_tell).
```
[the remark this answers is not in the document]
Comment (benjamc) [reply docs/advanced_usage/5_ask_and_tell.md]: fixed

File: docs/advanced_usage/9_parallelism.md [hunk 3]
```diff
@@ -1,42 +1,41 @@
...
     When using multiple workers, SMAC is not reproducible anymore.
 
 
-Running on a Cluster
---------------------
+## Running on a Cluster
+
 You can also pass a custom dask client, e.g. to run on a slurm cluster.
-See our :ref:`parallelism example<Parallelization-on-Cluster>`.
+See our [parallelism example](../../examples/1%20Basics/7_parallelization_cluster).
```
[the remark this answers is not in the document]
Comment (benjamc) [reply docs/advanced_usage/9_parallelism.md]: fixed

File: docs/advanced_usage/10_continue.md [hunk 4]
```diff
@@ -20,4 +19,4 @@ The behavior can be controlled by setting the parameter ``overwrite`` in the fac
     and the old run is not affected.
 
 
-Please have a look at our :ref:`continue example<Continue an Optimization>`.
\ No newline at end of file
+Please have a look at our [continue example](../../examples/1%20Basics/5_continue).
```
[the remark this answers is not in the document]
Comment (benjamc) [reply docs/advanced_usage/10_continue.md]: fixed

**Review by LukasFehring — APPROVED** [review]

**Pull request merged**